tests/Http/QueryTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Kirby\Http;
+
+use Kirby\TestCase;
+use PHPUnit\Framework\Attributes\CoversClass;
+
+#[CoversClass(Query::class)]
+class QueryTest extends TestCase
+{
+	public function testMerge(): void
+	{
+		$query = new Query(['foo' => 'bar']);
+		$query->merge(['baz' => 'qux']);
+		$this->assertSame('bar', $query->foo);
+		$this->assertSame('qux', $query->baz);
```

Comment:
```suggestion
		$params = new Query(['foo' => 'bar', 'bar' => 'baz']);
		$params->merge(['bar' => 'foo', 'baz' => 'qux']);
		$this->assertSame('bar', $params->foo);
		$this->assertSame('foo', $params->bar);
		$this->assertSame('qux', $params->baz);
```

Reply:
Included it manually when rebasing the PR.